assets/stylesheets/main.css

```diff
@@ -0,0 +1,1261 @@
+  background-color: rgb(51, 51, 51);
+  bottom: 0;
+  width: 100%;
+  z-index: 99999;
+}
+
+.subscribe .inner {
+  height: 40px;
+  max-width: 1200px;
+  padding: 0 20px;
+  margin: auto;
+  align-self: center;
+  color: white;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  position: relative;
+}
+
+.subscribe .inner p {
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 15px;
+  margin: 0;
+  position: absolute;
+  width: 100%;
+  padding-right: 40px;
+  box-sizing: border-box;
+  text-align: center
+}
+
+.subscribe .inner div span {
+  font-size: 14px;
+  font-weight: bold;
+  padding: 12.5px;
+}
+
+.subscribe .inner div {
+  margin-left: auto;
+  align-items: center;
+  display: flex;
+}
+
+.subscribe .inner svg {
+  width: 15px;
+  padding: 12.5px;
+}
+
+/* === */
+
+
+/* Nav */
+nav {
+  border-top: 2px solid rgb(226, 226, 226);
+  max-width: 1200px;
+  margin:auto;
+  padding-bottom: 9px;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+   
+}
+
+nav header{
+  border-top: 1px solid rgb(226, 226, 226);
+  height: 44px;
+  padding: 10px 0;
+  margin-top: 3px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav header a{
+  color: #6288a5;
+  font-size: 11px;
+
+}
+
+nav header svg{
+  height: 24.2px;
+}
+
+nav svg{
+  height: 13px;
+  margin-right: 7px;  
+}
+
+nav> div{
+  display: flex;
+  justify-content: space-between;
+  color: rgb(51, 51, 51);
+}
+
+nav h3{
+  text-transform: uppercase;
+  color: rgb(51, 51, 51);
+  font-size: 13px;
+  margin:0;
+  height: 25px;
+}
+
+nav ul{
+  padding: 0;
+  margin: 0;
+}
+
+nav ul li{
+  text-transform: capitalize;
+  font-size: 14px;
+  line-height: 16px;
+  height: 23px;
+
+}
+
+nav > div > div{
+  padding-left: 15px;
+  border-left: 1px solid rgb(228, 228, 228);
+}
+
+nav div div ul li{
+  font-size: 11px;
+  margin-bottom: 5px;
+}
+
+nav div div ul :nth-child(3){
+  border-bottom: 1px solid rgb(228, 228, 228);
+  padding-bottom: 5px;
+  margin-bottom: 10px;
+}
+
+nav div div div{
+  margin-bottom: 15px;
+}
+
+nav div div div ul li {
+  font-size: 16px;
+  font-weight: bold;
+  height: 25px; 
+}
+
+/* ====== */
+
+/* Footer */
+
+footer{
+  display: flex;
+  font-size: 11px;
+  padding: 10px 20px ; 
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+   border-top: 1px solid;
+   border-top-color: rgb(228, 228, 228);
+   max-width: 1200px;
+   margin:auto;
+   justify-content: center;
+}
+
+footer ul{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+footer ul{
+  margin:0;
+  padding: 0;
+}
+
+footer :nth-child(2) li{
+  padding:0 9px;
+}
+
+footer ul li a{
+  color: rgb(151, 151, 151);
+}
+
+
+
+/* ===== */
+
+
+/* Responsivness media query */
+
+@media only screen and (max-width: 1400px) {
+  article header figure {
+    width: 945px;
+    margin-left: auto;
+  }
+  aside{
+    display: none;
```

Comment:
Because that I don't see the aside part of the HTML file. When I comment it, it appears, please check that and solve it.

We must see that

Reply:
You can see the aside element at any screen width wider than 1400px, I will change it to be triggered under 1240px instead.